src/codeflare_sdk/cluster/config.py

```diff
@@ -53,6 +53,7 @@ class ClusterConfiguration:
     write_to_file: bool = False
     verify_tls: bool = True
     labels: dict = field(default_factory=dict)
+    persistent_volumes: dict = field(default_factory=list)
```

Comment:
Could this directly be passed as a two args, `persistent_volume_claims` and `volume_mounts`, and use k8 python objects directly? (`V1Volume` and `V1VolumeMount`) My gut feeling is that this would save some headache in the future when more features are asked for around volumes (i.e. mounting secrets). If we do this it puts a bit more onus on the user, but most users should be fairly familiar with the python k8 api. WDYT?

Reply:
Yeah, I think that would be good. I would basically just go based off of this documentation. https://kubernetes.io/docs/concepts/storage/volumes. I guess it makes more sense for it to be `volumes` and `volume_mounts` now that I think about it a bit more since pvc is just a particular type of volume that you can specify